doc/src/build/move/build-test.md

```diff
@@ -17,6 +17,8 @@ Build Successful
 Artifacts path: "./build"
 ```
 
+If the build fails, you will be presented with verbose error messaging you can use to troubleshoot and resolve root issues.

```

Comment:
I think IBM style suggests present tense when possible and avoid passive voice (you are presented). Perhaps 
If the build fails, you can use the verbose error messaging returned to resolve...

Reply:
Responding to @Clay-Mysten's question from the other PR (https://github.com/MystenLabs/sui/pull/4265#issuecomment-1226563062) - this feels a bit redundant to me but the example we give right above shows no output so augmenting this with and additional line stating that the output will be different in case of errors may be a good idea after all.

I am OK with it either way - I'd say go with what reads better for you @Clay-Mysten and @randall-Mysten

Reply:
Thank you both!